persistence/src/main/java/edu/unc/lib/dl/persist/services/delete/MarkForDeletionService.java

```diff
@@ -41,11 +46,19 @@
      * @param pids pids of objects to mark for deletion
      */
     public void markForDeletion(AgentPrincipals agent, String... ids) {
+        Collection<PID> removed = new ArrayList<>();
+        PID pid = null;
         for (String id : ids) {
-            PID pid = PIDs.get(id);
-            Runnable job = new MarkForDeletionJob(pid, agent,
-                    repositoryObjectLoader, sparqlUpdateService, aclService);
+            pid = PIDs.get(id);
+            Runnable job = new MarkForDeletionJob(pid, agent, repositoryObjectLoader,
+                    sparqlUpdateService, aclService);
             job.run();
+
+            removed.add(pid);
+        }
+        if (pid != null) {
+            PID destination = repositoryObjectLoader.getRepositoryObject(pid).getParent().getPid();
+            operationsMessageSender.sendRemoveOperation(agent.getUsername(), destination, removed);
```

Comment:
Hmm, the Remove event has actually been used for destroy operations in the past, so actual removal of the object from the repo rather than marking for deletion.  Looking at master, there does not appear to be an existing event for this.  I would suggest making a new MarkForDeletion operation.  
Additionally, the event doesn't impact hierarchy (nothing moves or is removed), so I would suggest that the event not take a param for parents, just the updated objects

Reply:
i think i fixed this. let me know if you still see there are changes that need to be made